{"name": "test_add_course_except_case[case1]", "status": "passed", "description": "反向用例:加课失败的用例", "attachments": [{"name": "log", "source": "54c1dae5-abf3-489b-b357-78ee5a66a4c6-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'course_code': '000000', 'expected': '该加课码不存在或者已经失效'}"}], "start": 1614088503116, "stop": 1614088509418, "uuid": "18158e79-be41-49cf-b97f-e305b199bf8a", "historyId": "2ebfb9d628cb80e40f1279ec0e9e398b", "testCaseId": "ec32c7bb5ba6251d942df3ff1edd7dca", "fullName": "testcases.test_join_the_course.TestJoinTheCourse#test_add_course_except_case", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_join_the_course"}, {"name": "subSuite", "value": "TestJoinTheCourse"}, {"name": "host", "value": "MuSen"}, {"name": "thread", "value": "4476-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_join_the_course"}]}